pulsar-broker/src/test/java/org/apache/pulsar/client/api/TopicReaderTest.java

```diff
@@ -70,6 +70,7 @@ public class TopicReaderTest extends ProducerConsumerBase {
     @BeforeClass
     @Override
     protected void setup() throws Exception {
+        this.conf.setBrokerDeduplicationEnabled(true);
```

Comment:
`producer.send(String.format("msg num %d", i).getBytes());`
the test is not sending duplicate message, why we need to dedup though?